src/writeSummaryToPR.ts

```diff
@@ -18,6 +19,11 @@ const writeSummaryToPR = async ({ summary, markerPostfix }: {
     pullRequestNumber = await getPullRequestNumberFromTriggeringWorkflow(octokit);
   }
 
+  if (!pullRequestNumber && userDefinedPrNumber) {
+    core.info(`User-defined pull-request number: ${userDefinedPrNumber}`);
+    pullRequestNumber = userDefinedPrNumber;
+  }
+
```

Comment:
Imho, the `userDefinedPrNumber` should take precedence over the automatically acquired ones. 

At least, that's how I would expect it if I specifically put the PR Number in. Could you adjust the order her accordingly? 🙂